<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRandom</class>
 <widget class="QDialog" name="DialogRandom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Pixel Image</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelButtonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="widthLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="widthLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="formatComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>32-bit ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24-bit RGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8-bit Grayscale</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="formatLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Format:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogRandom</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogRandom</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
